commit f2d29d2cefd2d92729f7af507243afbdd49a0304
Author: Dan Mick <dmick@redhat.com>
Date:   Fri Apr 28 17:57:52 2017 -0700

    mgr/PyModules.cc,PyState.cc: improve config access debug logging
    
    Signed-off-by: Dan Mick <dan.mick@redhat.com>

diff --git a/src/mgr/PyModules.cc b/src/mgr/PyModules.cc
index 10ac4adb8de..89347e2326b 100644
--- a/src/mgr/PyModules.cc
+++ b/src/mgr/PyModules.cc
@@ -543,6 +543,8 @@ bool PyModules::get_config(const std::string &handle,
 
   const std::string global_key = config_prefix + handle + "." + key;
 
+  dout(4) << __func__ << "key: " << global_key << dendl;
+
   if (config_cache.count(global_key)) {
     *val = config_cache.at(global_key);
     return true;
